kyo-core/shared/src/main/scala/kyo/KyoApp.scala

```diff
@@ -74,82 +62,12 @@ object KyoApp:
-
-        /** Runs an effect with a specified timeout, throwing an exception if it fails.
-          *
-          * Note: This method is unsafe and should only be used as the entrypoint of an application.
-          *
-          * @param timeout
-          *   The maximum duration to wait for the effect to complete.
-          * @param v
-          *   The effect to run.
-          * @param ev
-          *   Evidence that A is Flat.
-          * @param frame
-          *   The implicit Frame.
-          * @return
-          *   The computed value of type A.
-          * @throws Throwable
-          *   if the effect fails or times out.
-          */
-        def run[A: Flat](timeout: Duration)(v: A < Effects)(using Frame, AllowUnsafe): A =
-            attempt(timeout)(v).getOrThrow
-
-        /** Runs an effect with an infinite timeout.
-          *
-          * Note: This method is unsafe and should only be used as the entrypoint of an application.
-          *
-          * @param v
-          *   The effect to run.
-          * @param ev
-          *   Evidence that A is Flat.
-          * @param frame
-          *   The implicit Frame.
-          * @return
-          *   The computed value of type A.
-          * @throws Throwable
-          *   if the effect fails.
-          */
-        def run[A: Flat](v: A < Effects)(using Frame, AllowUnsafe): A =
-            run(Duration.Infinity)(v)
-
-        /** Creates a Fiber to run an effect with an infinite timeout.
-          *
-          * Note: This method is unsafe and should only be used as the entrypoint of an application.
-          *
-          * @param v
-          *   The effect to run.
-          * @param ev
-          *   Evidence that A is Flat.
-          * @param frame
-          *   The implicit Frame.
-          * @return
-          *   A Fiber representing the running effect.
-          */
-        def runFiber[A: Flat](v: A < Effects)(using Frame, AllowUnsafe): Fiber[Throwable, A] =
-            runFiber(Duration.Infinity)(v)
-
-        /** Creates a Fiber to run an effect with a specified timeout.
-          *
-          * Note: This method is unsafe and should only be used as the entrypoint of an application.
-          *
-          * @param timeout
-          *   The maximum duration to wait for the effect to complete.
-          * @param v
-          *   The effect to run.
-          * @param ev
-          *   Evidence that A is Flat.
-          * @param frame
-          *   The implicit Frame.
-          * @return
-          *   A Fiber representing the running effect.
+          *   A Result containing either the computed value or a failure.
           */
-        def runFiber[A: Flat](timeout: Duration)(v: A < Effects)(using Frame, AllowUnsafe): Fiber[Throwable, A] =
-            v.pipe(Resource.run, Async.timeout(timeout), Async.run, IO.Unsafe.run).eval
+        def runAndBlock[A: Flat](timeout: Duration)(
```

Comment:
Simplified all methods to a single one that indicates it's blocking (something important that was missing) and returns a `Result`. The other methods didn't seem very useful and their behavior can be implemented by the user via regular effect handling before calling `runAndBlock`.